packages/client/src/comp/states/BaseContainer.ts

```diff
@@ -1,19 +1,24 @@
 import React from 'react'
 
 import { createContainer } from 'unstated-next'
+import { AddressInputId } from './AddressFormContainer'
 
 const datePattern = /(0[1-9]|1[012])[/.](0[1-9]|[12][0-9]|3[01])[/.](19|20)[0-9]{2}/i
 // eslint-disable-next-line no-useless-escape
 const emailPattern = /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i
 const telephonePattern = /[0-9]{3}-?[0-9]{3}-?[0-9]{4}/
 
-type InputId =
+export type InputId =
```

Comment:
I think we can reuse the AddressFormContainer here under the value `mailingAddress`